src/libfsm/determinise.c

```diff
@@ -516,12 +632,19 @@ determinise(struct fsm *nfa,
 	}
 
 	/*
-	 * While there are still DFA states remaining to be "done", process each.
+	 * While there are still states remaining to be "done", process each.
 	 */
 	sv = sv_init;
 	for (curr = mapping_set_first(mappings, &it); (curr = nextnotdone(mappings, &sv)) != NULL; curr->done = 1) {
-		if (!buildtransitions(nfa, dfa, mappings, curr)) {
-			goto error;
+		/* XXX - may want to hoist this outside the loop to avoid branching */
```

Comment:
Probably not, our bottlenecks are very much elsewhere

Reply:
removed comment: 0e11a68